src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -664,7 +761,7 @@ private void SetValueForTypedDatabaseCommand(TransactionOperationContext context
                 updateCommand = (UpdateValueForDatabaseCommand)JsonDeserializationCluster.Commands[type](cmd);
 
                 var record = ReadDatabase(context, updateCommand.DatabaseName);
-                if (record == null)
+                if (record == null && updateCommand.FromBackup == false)
```

Comment:
Why? This shouldn't be here. We should't be setting values before the db record exists

Reply:
fixed